### ISSUE TITLE
Dandelion singularity (v0.2.4) cannot quantify mutation

### ISSUE THREAD

**Issue by sbenjamaporn** [zktuong/dandelion#313]
### Description of the bug

Dear zktuong,

I have used this singularity for processing step which they were sucessful in many times ago.
Now, I am facing with this problem that the singularity cannot quantify mutation as attached figure.

If you have any suggestions, please let me know
![E0C0153F-3953-46C0-B2E3-41D021A5AAC6_1_105_c](https://github.com/zktuong/dandelion/assets/92794713/55a08fdf-f817-4df2-a3c2-e715546af0ac)

Best regards,
Benjamaporn

### Version information

dandelion==0.2.4 pandas==1.4.3 numpy==1.22.4 matplotlib==3.5.2 networkx==2.8.4 scipy==1.8.1

**Comment by zktuong** [comment]
hi @sbenjamaporn, not sure what's happened there but it looks like your singularity image decided to pull in your R from your local environment (outside the container). 

maybe can you try to update your image and try again?

**Comment by sbenjamaporn** [comment]
@zktuong Thanks for your suggestion,

Yes, I worked on R and found that it might relate with my local enviroment as you mentioned.
I changed the location of the singularity in another private server, and it can be sucessefully pre-processed !.
However, I got the warning messages from R as attached images. 
Do you think it possibly affect my results or it just the warmings ?  (as I asked this question in the earlier discussion: https://github.com/zktuong/dandelion/issues/156#issuecomment-1193119247)
![E2D888DD-6493-4B7B-B6E4-82EBBA12652B]
(https://github.com/zktuong/dandelion/assets/92794713/a4aa75c0-9173-4345-ab25-bf6ccbde496a)

Best regards,
Benjamaporn

**Comment by zktuong** [comment]
Yup it’s just warnings and don’t need to worry about it.

**Comment by sbenjamaporn** [comment]
@zktuong I got it!
 
Becase I did not use the command "singularity run -B $PWD"  in the pre-processing step (I go to the sample folder, paste, and then run "sc-dandelion_latest.sif dandelion-preprocess --meta meta.csv" , so it use R from my local enviroment. However, I don't think it will impact my result since it show "Pre-processing finished". Do I understand this step correctly ?

For the next analysis, I will use "singularity run -B $PWD" to stop the warmings

Benjamaporn
